modules/ai/assets/js/editor/pages/form-layout/hooks/use-screenshots.js

```diff
@@ -59,6 +71,12 @@ const useScreenshots = ( { onData } ) => {
 				prompt,
 				prevGeneratedIds,
 				currentContext,
+				ids: {
+					sessionId,
+					generateId: generateIdRef.current,
+					batchId,
+					requestId: `request-${ getUniqueId() }`,
```

Comment:
why those IDs are being handled here and not in the backend?
If I understand correctly, seems like the only client-dependant ID is the session, am I right?

Reply:
The backend stateless, so it's blind to the batch to a regenerate.